### ISSUE TITLE
Wrong system bar color on iOS when using safari

### ISSUE THREAD

**Issue by lordfido** [GoogleChromeLabs/pwacompat#10]
I'm using `"theme_color": "#191919"`, wich I consider a dark color. This bug can be reproduced when using `#000000` also. 

Then, I'm adding my PWA to my homescreen and the iOS  system bar is being painted on white, instead of black.

If I change the `theme_color` value on my manifest to `#FFFFFF`, then the iOS system bar is painted on black. The opposite of my `theme_color`.

I've being looking into the code, and the problem is on [pwacompat.js:276](https://github.com/GoogleChromeLabs/pwacompat/blob/master/pwacompat.js#L276), where I think a change should be made:

```
-    meta('apple-mobile-web-app-status-bar-style', themeIsLight ? 'default' : 'black');
+    meta('apple-mobile-web-app-status-bar-style', themeIsLight ? 'black' : 'default');
```

This would be also coherent with the same feature for MS Edge (on [pwacompat.js:284](https://github.com/GoogleChromeLabs/pwacompat/blob/master/pwacompat.js#L284))

**Comment by lordfido** [comment]
I've created a Pull request https://github.com/GoogleChromeLabs/pwacompat/pull/11

**Comment by samthor** [comment]
Fixed by #11.
